### PR TITLE
webpage: add missing titles + more badge previews

### PR DESCRIPTION
**Pull request by tooomm**
`Node 9 support` (#1290)
over
`Github rate limits cause transient service test failures in CI` (#979)

Was messing around with the layout due to the very wide column needed:
untitled

~~Follow-up commit will fix tests (editing from web).~~

### CODE REVIEW

**Comment by tooomm** [comment]
Shouldn't the first commit fail? :)

**Comment by paulmelnikow** [comment]
I think this was already fixed in #1297. You can see what it looks like here:
https://shields-staging.herokuapp.com/

The tests and the example page are independent. They don't need to match.

**Comment by tooomm** [comment]
From https://github.com/badges/shields/pull/1297#pullrequestreview-80315164
>This badge is really wide. Omitting previewUri prevents the badge from displaying and growing the column too wide.

Isn't that more of a workaround instead a fix? ;)
Not having a preview at all feels bad. That's a very important part of the shields site, no?

From shields-staging.herokuapp.com:
preview
**(There are several badges having no description/headline, too!)**

<br>

>The tests and the example page are independent. They don't need to match.

Makes sense, the test itself is still valid. For me it sounds more logical to match them though.

**Comment by paulmelnikow** [comment]
Eh, it's the same workaround as several of the other long badges.

I do think they should all have a title / headline…

**Comment by tooomm** [comment]
>Eh, it's the same workaround as several of the other long badges.

`TeamCity (full build status)` is the only other one which seems to be missing a badge preview.
Problem there is that you can't even enter preview mode at all if you try to click the example link.

>I do think they should all have a title / headline…

Tried to fix that. Hope I hit all. Please double check, or could you deploy to Heroku review apps?

File: lib/all-badge-examples.js [hunk 1]
```diff
@@ -2046,15 +2046,15 @@ const allBadgeExamples = [
...
       },
       {
-        title: 'Issue Stats',
+        title: 'Pull Request Stats',
         previewUri: '/issuestats/p/github/expressjs/express.svg'
       },
       {
-        title: '(long form)',
+        title: 'Pull Request Stats (long form)',
         exampleUri: '/issuestats/p/long/github/expressjs/express.svg'
```
Comment (tooomm) [inline lib/all-badge-examples.js]: I changed the title here to match what the badge is showing. 

Example badge shows: `pull closure | 16 hrs`

**Comment by paulmelnikow** [comment]
> `TeamCity (full build status)` is the only other one which seems to be missing a badge preview.
Problem there is that you can't even enter preview mode at all if you try to click the example link.

Hmm, it seems to be working on current staging:

screen shot 2017-12-01 at 5 42 24 pm

**Comment by tooomm** [comment]
Not for me:
untitled

Clicking on either the name, the non-existant preview or the example link on the right doesn't open that window.
**Actually it's not working for all other badges too! (Have a look at the shortened links all are displaying!)**

**Comment by paulmelnikow** [comment]
Same here on the PR branch. You might try merging master into this one. Probably one of my recent commits fixed it.

**Comment by tooomm** [comment]
Can I request you to do so, please @paulmelnikow ?
I can only use the web interface right now.

**Comment by tooomm** [comment]
>Working: https://shields-staging.herokuapp.com
Not working: https://shields-staging-pr-1305.herokuapp.com

I'm sorry, but both have the same problem for me. Even after merging most recent badges/shields master to my fork (and this branch).
Neither open the [preview window where you can do adjustments](https://github.com/badges/shields/pull/1305#issuecomment-348633925) to the badge.

**Comment by tooomm** [comment]
No empty titles anymore and all preview badges show (except that very big one for `TeamCity (full build status)`.

Last several comments are basically not relevant anymore due to https://github.com/badges/shields/pull/1322.

Can you revisit this @paulmelnikow?

**Comment by paulmelnikow** [comment]
I'm fine with the badge changes, though the test change doesn't seem necessary. It's not necessary to keep them perfectly in sync.

**Pull request merged**